src/tests/end2end/ReduceTests.cpp

```diff
@@ -0,0 +1,716 @@
+// Copyright 2021 The WebNN-native Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "src/tests/WebnnTest.h"
+
+enum ReduceOpType {
```

Comment:
Move to L.28
```
private:
     enum ReduceType {
        kReduceL1 = 0,
         .....
     }
```

Reply:
CheckReduce() function can't access private ReduceType, how about :
`protected:
     enum ReduceType {
        kReduceL1 = 0,
         .....
     }`

Reply:
Ok, please use `protected`